packages/apollo-tracing/src/index.ts

```diff
@@ -33,6 +33,97 @@ interface ResolverCall {
   endOffset?: HighResolutionTime;
 }
 
+export const plugin = (_futureOptions = {}) => (): ApolloServerPlugin => ({
+  requestDidStart() {
+    let startWallTime: Date | undefined;
+    let endWallTime: Date | undefined;
+    let startHrTime: HighResolutionTime | undefined;
+    let duration: HighResolutionTime | undefined;
+    const resolverCalls: ResolverCall[] = [];
+
+    startWallTime = new Date();
+    startHrTime = process.hrtime();
```

Comment:
Is tracing intended to include timing for parse/validate? I would've imagined these happening within `executionDidStart`.

Reply:
Neither tracing or engine reporting really highlight the parsing and validation phases.  I suspect there is some outstanding intention to provide that insight eventually and that those would leverage this `startHrTime` as offsets to their timings, much in the way that this `startHrTime` is used as offsets for individual resolver timings right now.

For now, this maintains identical behavior to the existing behavior so we're not bringing any breaking changes along with this critical-to-unblock-3.x re-factor.  (We need to be careful to